src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,226 @@
+# Motion
+
+Support for Motion was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+For information on how to cite this extension when referencing it in the context of the academic literature, please read [Citing BIDS](../01-introduction.md#citing-bids).
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+This specification has been used to format a number of [example Motion datasets](https://github.com/bids-standard/bids-examples), which can be used as helpful guidance when curating new datasets.
+
+## Motion recording data
+
+{{ MACROS\_\__make_filename_template(
+"raw",
+datatypes=\["motion"],
+suffixes=\["motion", "channels", "events"])
+}}
+
```

Comment:
```suggestion
<!--
This block generates a filename templates.
The inputs for this macro can be found in the directory
  src/schema/rules/files/raw
and a guide for using macros can be found at
 https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
-->
{{ MACROS___make_filename_template(
    "raw",
    datatypes=["motion"],
    suffixes=["motion", "channels", "events"])
}}

```

@JuliusWelzel I'm sorry for the delay. Your macro call looks pretty good, but you don't need backslashes. We also recommend prefacing any macro calls with the comment I've suggested, to make the markdown files more readable to contributors.

Reply:
Thanks for the help! I start drafting the other macros

Reply:
@JuliusWelzel this is something you could address in the meantime